### ISSUE TITLE
Can not access Tekton menu items using the keyboard with Firefox

### ISSUE THREAD

**Issue by rnapoles-tktn** [tektoncd/dashboard#900]
**Problem:**
When using Firefox (68.4.1esr (64-bit) on MacOS) I can not navigate the Tekton left menu using the keyboard.
(Works as expected on Chrome)

**Expected Result:**
User should be able to navigate the UI using the keyboard. 

**Steps to reproduce:**
Use Firefox 68.4.1esr (64-bit) on MacOS to navigate to the Tekton homepage. Use the tab key to try to select all the left side menu options.
The focus jumps from the top Tekton collapsable menu to the "All Namespaces" dropdown, then it will jump to the elements in the displayed right pane skipping all menu options under Tekton (Pipelines, PipelineRuns, PipelineResources, Task, ClusterTasks, TaskRuns). It also skips Import Tekton resources, Secrets and Webhooks.

**Comment by chmouel** [comment]
Or nevermind, @vdemeester kindly had the rights to transfer your issue on `tektoncd/dashboard`

**Comment by a-roberts** [comment]
Made a new accessibility label but really I think this is about having the dashboard well tested using Selenium (@rnapoles-tktn mentioned this on our daily scrum calls at IBM we have), I'm curious if this is the case in other browsers too so it'll be great for anyone to pick this up

**Comment by AlanGreene** [comment]
@rnapoles-tktn I can't reproduce this in either Firefox ESR (68.4.1) or latest (72.0.1). In both cases the tab key correctly moves through all of the left nav items, including Import Tekton Resources etc.

The left nav is based on the Carbon SideNav component so I'd be curious to know if you have the same issue on their storybook for this component. This might help us narrow down your issue. http://react.carbondesignsystem.com/?path=/story/ui-shell--header-base-w-sidenav

**Comment by rnapoles-tktn** [comment]
@AlanGreene I tried on the storybook for the select component and I can navigate through the options using the same browser with which I have the problem in Tekton, however, when I inspect the code, the select in tekton is not implemented the same way as in the storybook.
Tekton:
![image](https://user-images.githubusercontent.com/57016183/72563273-4ea83200-387b-11ea-9544-bf209743796c.png)

Carbon storybook:
![image](https://user-images.githubusercontent.com/57016183/72563395-9333cd80-387b-11ea-98ae-3991709461de.png)

**Comment by AlanGreene** [comment]
@rnapoles-tktn the Tekton screenshot in your comment shows the SideNav menu and menu items, the Carbon screenshot shows the Select component. These are unrelated components.

The dashboard uses the [carbon-components-react](https://www.npmjs.com/package/carbon-components-react) component library which is published by the Carbon team, so I would expect the DOM structure of the basic components to very closely match what you find in the Carbon storybook. Where there are discrepancies these are most likely due to customisations / optional features you can often find and control in the knobs tab of the storybook, or documented in the info panel.

![image](https://user-images.githubusercontent.com/2829095/72756986-b8e80c00-3bc6-11ea-9838-53bf14d8624c.png)


You can find examples of the SideNav component in the Carbon storybook. See http://react.carbondesignsystem.com/?path=/story/ui-shell--header-base-w-sidenav and other items referencing SideNav under the UI Shell menu in the left nav.

![image](https://user-images.githubusercontent.com/2829095/72756319-b84e7600-3bc4-11ea-88a1-919a9da621a7.png)


As for the issue you reported with keyboard navigation:
- Can you confirm this behaviour on the storybook I linked above?
- Which macOS version are you running?
- Are you using any browser extensions or other applications that might be affecting keyboard use?
  - Try reproducing in a private window with extensions disabled
